packages/react-components/react-link/library/src/components/Link/useLink.ts

```diff
@@ -31,7 +33,7 @@ export const useLink_unstable = (
     appearance,
     disabled,
     disabledFocusable,
-    inline,
+    inline: inline || !!inlineContext,
```

Comment:
```suggestion
    inline: inline ?? !!inlineContext,
```

As we want to allow `inline={false}`